packages/@okta/vuepress-theme-default/components/GuidesOverview.vue

```diff
@@ -3,8 +3,7 @@
     <GuidesNavigation />
     <div class="overview-content">
       <h1>Guides</h1>
-      <p>Step-by-step guides for your use cases.</p>
-      <p>With Okta, you can lorem like no ipsum before</p>
+      <Content />
```

Comment:
Thus resolves https://oktainc.atlassian.net/browse/OKTA-219919

Reply:
LOL! I didn't realize that had been logged. 👍